en/02_Developer_Guides/09_Security/04_Sudo_Mode.md

```diff
@@ -19,39 +38,24 @@ SilverStripe\Security\SudoMode\SudoModeService:
   lifetime_minutes: 25
 ```
 
-## Enabling sudo mode for controllers
+## Enabling sudo mode for controller endpoints
```

Comment:
Just as a sanity check in case you didn't already - please quickly check if there are any links in the docs to the old anchor, and update them to the new anchor if there are.

Reply:
Checked and did not find any